h2o-core/src/main/java/hex/LinkFunction.java

```diff
@@ -0,0 +1,95 @@
+package hex;
+
+import hex.LogExpUtil;
+
+import java.io.Serializable;
+
+abstract class LinkFunction implements Serializable {
```

Comment:
Do not use Serializable, extend from Iced instead. Serializable will work (most of the time) but it will be less efficient. Please prefer Iced whenever possible.

Reply:
Ok, thank you.